orttraining/orttraining/test/python/_test_helpers.py

```diff
@@ -194,10 +195,11 @@ def assert_values_are_close(input, other, rtol=1e-05, atol=1e-06):
         assert False, err_msg
 
 def enable_custom_autograd_function(module):
-    for mode in [True, False]:
-        module._torch_module._execution_manager(mode)._enable_custom_autograd_function = True
-        module._torch_module._execution_manager(mode)._save_onnx = True
-        module._torch_module._execution_manager(mode)._save_onnx_prefix = "utbench"
+    # load from json.
+    path_to_json = os.path.join(os.getcwd(), 'orttraining_test_ortmodule_autograd_config.json')
+    load_from_json(module, path_to_json)
```

Comment:
load from json is an experimental feature which can be deprecated. Relying on experimental code for a production feature is not adviced

Reply:
Then how should we enable the feature without load from json?